docs/standard/datetime/access-utc-and-local.md

```diff
@@ -6,12 +6,12 @@ dev_langs:
   - "csharp"
   - "vb"
 helpviewer_keywords: 
-  - "time zones [.NET Framework], local"
+  - "time zones [.NET], local"
   - "predefined time zones"
   - "UTC times, predefined"
   - "local time zone access"
-  - "time zones [.NET Framework], retrieving"
-  - "time zones [.NET Framework], UTC"
+  - "time zones [.NET], retrieving"
+  - "time zones [.NET], UTC"
```

Comment:
These are used for SEO purposes right?
In that case, I'd extend them instead of updating them.
i.e, if someone searched for "Time zones in .NET Framework" in a search engine (Google, Bing, etc), this change may decrease visibility if I understand this correctly.

Reply:
No, these are purely for the index in the offline help viewer app.